### PR TITLE
update angular-embed to version for angular 1.6

### PR DESCRIPTION
**Pull request by petrjasek**
SDESK-1063

### CODE REVIEW

File: scripts/core/editor2/add-embed.ctrl.js [hunk 1]
```diff
@@ -1,6 +1,16 @@
 var embedCodeHandlers = require('./embedCodeHandlers.js');
 
-angular.module('superdesk.apps.editor2.embed', []).controller('SdAddEmbedController', SdAddEmbedController);
+angular.module('superdesk.apps.editor2.embed', []).controller('SdAddEmbedController', SdAddEmbedController)
+    .config(['$sceDelegateProvider', ($sceDelegateProvider) => {
+        $sceDelegateProvider.resourceUrlWhitelist([
```
Comment (naspeh) [inline scripts/core/editor2/add-embed.ctrl.js]: this list repeats 2 times in https://github.com/superdesk/angular-embed/pull/5/files and here ones

maybe it should be in variable, what do you think?

File: scripts/core/editor2/editor.js [hunk 2]
```diff
@@ -698,7 +698,8 @@ angular.module('superdesk.apps.editor2', [
     'superdesk.apps.editor2.utils',
     'superdesk.apps.spellcheck',
     'superdesk.apps.authoring',
-    'angular-embed'
+    'angular-embed',
+    'angular-embed.handlers'
```
Comment (gbbr) [inline scripts/core/editor2/editor.js]: This seems unusual. Should `angular-embed` not automatically include `angular-embed.handlers`?

Comment (petrjasek) [inline scripts/core/editor2/editor.js]: it seems to be the other way around https://github.com/superdesk/angular-embed/blob/master/dist/angular-embed.js#L28

File: scripts/core/editor2/add-embed.ctrl.js (hunk 1 above)
Comment (petrjasek) [inline scripts/core/editor2/add-embed.ctrl.js]: I've removed it from there, should be enough to have it in lib

File: package.json [hunk 3]
```diff
@@ -33,7 +33,7 @@
     "angular-bootstrap-npm": "jusopi/angular-bootstrap-npm#0.13.1",
     "angular-contenteditable": "0.0.2",
     "angular-dynamic-locale": "^0.1.32",
-    "angular-embed": "superdesk/angular-embed#2afd7397a7179f25b66c4b5f13ea2a0daf799821",
+    "angular-embed": "petrjasek/angular-embed#79c1038",
```
Comment (naspeh) [inline package.json]: I'll change this after merging PR in `superdesk/angular-embed`, right?

Comment (petrjasek) [inline package.json]: updated it

Comment (naspeh) [inline package.json]: great! BTW: I checked on http://sdcpr-1508.test.superdesk.org/ instance and it's working now :)

File: scripts/core/editor2/editor.js (hunk 2 above)
Comment (gbbr) [inline scripts/core/editor2/editor.js]: Maybe we should have only one then... ?

Comment (petrjasek) [inline scripts/core/editor2/editor.js]: I've removed the later, it wasn't there before and it is not needed for the fix

File: package.json [hunk 4]
```diff
@@ -127,6 +127,7 @@
     "karma-webpack": "~1.8.1",
     "protractor": "~4.0.14",
     "react-addons-test-utils": "~15.5.1",
+    "react-test-renderer": "~15.5.1",
```
Comment (naspeh) [inline package.json]: this the fix for `npm test`, right?

Comment (petrjasek) [inline package.json]: yes

**Pull request merged**